src/index.html

```diff
@@ -91,10 +93,23 @@
   </paper-dialog>
 
   <!-- Report dialog -->
-  <paper-dialog id="report-dialog" heading="Bug Report" backdrop autoCloseDisabled transition="paper-dialog-transition-center">
-    <p>Please file a bug in the <a id="report-link">Chromium bug tracker</a> and attach the report from this page.</p>
-    <paper-button id="report-download" onclick="report.downloadReport()">Download Report</paper-button>
-    <paper-button onclick="this.parentElement.close()">Close</paper-button>
+  <paper-dialog id="report-bug-dialog" heading="File a bug" autoCloseDisabled backdrop transition="paper-dialog-transition-center">
+    <paper-input-decorator floatingLabel label="Describe your issue here">
+      <paper-autogrow-textarea id="bug-description-area">
+        <textarea id="bug-description"></textarea>
+      </paper-autogrow-textarea>
+    </paper-input-decorator>
+    <h3>Disclaimer</h3>
+    <p>Submitting this bug report will generate a log file (deleted after 90 days) containing information about your device which will be uploaded to the WebRTC troubleshooter server and will be used for troubleshooting. Note that the log file does not contain any personal information.</p>
+    <paper-button raised id="upload-button" onclick="report.prepareReport(this.  parentElement)">Submit bug report</paper-button>
```

Comment:
My editor wraps the code if it exceeds the view port hiding spaces and since we do not do HTML linting yet grunt did not catch it either.
Spaces removed.